go/vt/vtctl/workflow/resharder.go

```diff
@@ -99,6 +99,9 @@ func (s *Server) buildResharder(ctx context.Context, keyspace, workflow string,
 		if err != nil {
 			return nil, vterrors.Wrapf(err, "GetShard(%s) failed", shard)
 		}
+		if si.PrimaryAlias == nil {
+			return nil, fmt.Errorf("target shard %v has no primary tablet", shard)
+		}
```

Comment:
This is unrelated, but addresses a panic raised in the Vitess Slack where the target shard had no primary tablet.